sdk/tables/azure-data-tables/azure/data/tables/_table_client.py

```diff
@@ -34,9 +34,19 @@
 class TableClient(TablesBaseClient):
     """A client to interact with a specific Table in an Azure Tables account.
 
-    :ivar str account_name: The name of the Tables account.
-    :ivar str table_name: The name of the table.
-    :ivar str url: The full URL to the Tables account.
```

Comment:
Why did we remove the `:ivar:` documentation? This should be maintained separate to the constructor params.

Reply:
Actually the constructor params are already documented on line 60 with the constructor - why are we duplicating it here?

Reply:
> Why did we remove the `:ivar:` documentation? This should be maintained separate to the constructor params.

It's good to know. I found `account_name` in parent class, I'll add it back. But I think the rest are not accurate: - `table_name` is a positional argument, so it should be `:param str table_name`, - I don't think `url` is a variable in this class, I found variable `scheme` instead.

Reply:
> Actually the constructor params are already documented on line 60 with the constructor - why are we duplicating it here?

Idk. I found there are some class having duplicate docstring in class and constructor. I tried with next pylint, it's happy to have either one.

Reply:
For the case that having `:ivars`, `:params` and `:keyword`, I'll remove class level docstring but the description for `:ivars` and keep constructor docstring. For the case just having `:params` and `:keyword`, the class level and constructor level docstrings should be identical, I'll only keep either one.

Reply:
As mentioned above - we still need to include the `@properties` which are `url` and `api_version`